routes/matches/matches.js

```diff
@@ -775,15 +815,15 @@ router.get("/limit/:limiter", async (req, res, next) => {
     let sql;
     if (req.user !== undefined && Utils.superAdminCheck(req.user)) {
       sql =
-        "SELECT * FROM `match` WHERE cancelled = 0 OR cancelled IS NULL ORDER BY id DESC LIMIT ?,?";
+        "SELECT mtch.* FROM `match` mtch WHERE mtch.cancelled = 0 OR mtch.cancelled IS NULL ORDER BY mtch.id DESC LIMIT ?,?";
```

Comment:
Does this really need to be changed if you haven't made any changes to it aside from adding on the identifier when it's not needed?